<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Forgot_password</class>
 <widget class="QDialog" name="Forgot_password">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Forgot-Password</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 170, 255);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>381</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
 border: 2px solid purple;
    border-radius: 10px;
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="Update">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: purple;
    
	color: rgb(255, 255, 255);
    border: 1px solid black;
    padding: 5px 10px;
font: 87 10pt &quot;Arial Black&quot;;

}

QPushButton:hover {
    background-color: red;
}

QPushButton:pressed {
    background-color: green;
    color: white;  /* Text color when pressed */
	
}
</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>381</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true"> QPushButton{

background: transparent;
 border: none;

	font: 12pt &quot;Algerian&quot;;
color: rgb(85, 0, 255);
}

QPushButton:pressed
 { background: green;
 color: black; font:
 bold 10px;
 }</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/Image/key (1).png</normaloff>:/Image/key (1).png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">  background-color: purple;
font: 87 14pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);
border:none;</string>
       </property>
       <property name="text">
        <string>RESET PASSWORD</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>140</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Algerian&quot;;
color: rgb(85, 0, 255);
border:none;</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameLineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>145</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {  border: 2px solid gray;
	font: 87 10pt &quot;Arial Black&quot;;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);

border-radius: 5px;}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>190</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Algerian&quot;;
color: rgb(85, 0, 255);
border:none;</string>
    </property>
    <property name="text">
     <string>New-password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newPasswordLineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {  border: 2px solid gray;
	font: 87 10pt &quot;Arial Black&quot;;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);

border-radius: 5px;}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirmPasswordLineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {  border: 2px solid gray;
	font: 87 10pt &quot;Arial Black&quot;;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);

border-radius: 5px;}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Algerian&quot;;
color: rgb(85, 0, 255);
border:none;</string>
    </property>
    <property name="text">
     <string>Confirm-pass</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
